docs/plugins.asciidoc

```diff
@@ -99,8 +99,7 @@ bin/kibana plugin --install username/sample-plugin --timeout 1m
 ==== Plugins and Custom Kibana Configurations
 
 Use the `-c` or `--config` options to specify the path to the configuration file used to start Kibana. By default, Kibana 
-uses the configuration file `config/kibana.yml`. When you change your installed plugins, the `bin/kibana plugin` command 
-restarts the Kibana server. When you are using a customized configuration file, you must specify the
+uses the configuration file `config/kibana.yml`. After you have changed the configuration file, restart the Kibana server using `service kibana stop` followed by `service kibana start`. When you are using a customized configuration file, you must specify the
```

Comment:
This is a good change, but I don't think we should mention the `service` command, since that only applies to specific systems with specific setups.